pkg/tools/gen/testdata/doc/pkg/md_got/docs/main.md

```diff
@@ -0,0 +1,76 @@
+# main
+
+## Index
+
+- [Container](#container)
+- [Server](#server)
+- k8s
+  - [Deployment](#deployment)
+  - core
+    - [PodSpec](#podspec)
+
+## Schemas
+
+### Container
+
+Container is the common user interface for long-running services.
+
+#### Attributes
+
+| name | type | description | default value |
+| --- | --- | --- | --- |
+|**image** `required`|str|||
+|**name** `required`|str|The name of the long-running container.||
+### Server
```

Comment:
There need to be some blank lines here, please check the modifications in pkg/tools/gen/templates/doc/schemaDoc.gotmpl